pages/recipes/[id]/index.js

```diff
@@ -60,6 +64,23 @@ export default function DetailsPage() {
       <StyledDetailsItem>
         <h4>How to prepare it:</h4> {recipe.instruction}
       </StyledDetailsItem>
+
+      <br />
+
+      <StyledLink $justifySelf="start" href={"/"}>
+        back
+      </StyledLink>
+      <div>
+        <StyledLink
+          href={`/recipes/${id}/edit`}
+          defaultData={recipe.ingredients}
+          passHref
+          legacyBehavior
+        >
+          <Link>Edit</Link>
+        </StyledLink>
```

Comment:
nowadays you don't need passHref and legacyBehavior anymore and don't need to wrap a Link in another Link, but can work directly like this:

```suggestion
        <StyledLink
          href={`/recipes/${id}/edit`}
          defaultData={recipe.ingredients}
        > 
        Edit
        </StyledLink>
```